CONTRIBUTING.md

```diff
@@ -15,79 +15,80 @@ highly appreciated!
 
 ## Development environment
 
-The development environment allows contributors to test their changes to the
-extension. The local repository source is built and loaded into the Extension
-Development Host.
+Setting up a dev environment allows contributors to test changes to this
+extension. The source code is built, and the resulting package loaded into a
+special instance of VS Code.
 
-First, ensure that [Yarn](https://yarnpkg.com/) and
-[Visual Studio Code](https://code.visualstudio.com) are installed.
+First, ensure that [Visual Studio Code](https://code.visualstudio.com) and
+[Yarn](https://yarnpkg.com/) are installed.
 
-- Open this repository's root directory as a project in VS Code.
+- Open this repository as a project in VS Code and run `yarn install` from the
+  _Integrated Terminal_.
 
-- Run `Tasks: Run Build Task` (Ctrl+Shift+b) to start the extension and webview
-  development servers (alternatively run `yarn dev-server` from the terminal).
+- Run `Tasks: Run Build Task` (Shift + Ctrl/Cmd + `b`) to start the extension
+  and Webview dev servers (alternatively, run `yarn dev-server` from the
+  _Terminal_). The latest build is saved in `extension/dist`.
 
-  > **Warning**: Having a separate (`.vsix`) version of the extension installed
-  > may cause all kinds of chaos in your development environment.
+  > **Warning**: Having a separate `.vsix` version of this extension installed
+  > may cause all kinds of chaos in your dev env.
 
-- Open the Extension Development Host, a child instance of VS Code with the
-  results of the dev servers installed as an extension, with `Start Debugging`
-  (f5).
+- **Start Debugging** (F5) to open the [Extension Development Host], a child
+  instance of VS Code with the local extension build installed.
 
   > **Note**: using the `Run Extension` command when running the debugger will
   > prevent all other extensions from loading into VS Code. This will improve
   > the performance of VS Code but can cause certain DVC commands to fail if the
-  > project uses an isolated python environment (see [this warning](#warning)).
+  > DVC project uses an isolated Python env (see [this warning](#warning)).
```

Comment:
Actually, this warning I'm also not getting: What is the `Run Extension` command?

<img width="955" alt="image" src="https://user-images.githubusercontent.com/1477535/176603417-bfd9f75a-6149-48b9-987c-633be6e4aa9e.png">

Reply:
The options are in a different place for the debugger: 
https://user-images.githubusercontent.com/37993418/176607460-3c1be14f-d168-4651-aa9f-c0045cf455d2.mov